### DISCUSSION TITLE
2 questions

### DISCUSSION THREAD

**Discussion by WXQ8NPb5TrbDNqzywcxF** [liedekef/events-made-easy#637] [General]
Hi !, 

Thanks for the plugin, very usefull.

2 questions for the "approved booking" section

1- Is it possible to add time when export to csv the "Date and Time" column ?  Actually it seem to only export date
2- Is it possible to add column "Location" in the selection ? I would like to show the location name of the event.

Thanks

**Comment by liedekef** [comment]
1) The "Date and Time" are exported with a newline between date and time, so make sure your program supports it :-)
2) Currently not possible, but you can export events (with their event id) and bookings (with their event id) and link them like that

**Comment by WXQ8NPb5TrbDNqzywcxF** [comment]
Thanks you,  I think i was blind for the time !
